.spelunky2/Locations.py

```diff
@@ -149,89 +149,89 @@ class Spelunky2LocationData(NamedTuple):
     "Grub Journal Entry": Spelunky2LocationData(111, "Sunken City"),
     "Giant Fly Journal Entry": Spelunky2LocationData(112, "Sunken City"),
     "Hundun Journal Entry": Spelunky2LocationData(113, "Sunken City"),
-    "Eggplant Minister Journal Entry": Spelunky2LocationData(114, "Sunken City"),
-    "Eggplup Journal Entry": Spelunky2LocationData(115, "Sunken City"),
+    "Eggplant Minister Journal Entry": Spelunky2LocationData(114, "Eggplant World"),
+    "Eggplup Journal Entry": Spelunky2LocationData(115, "Eggplant World"),
 
     # Cosmic Ocean
     "Celestial Jelly Journal Entry": Spelunky2LocationData(116, "Cosmic Ocean"),
 
     # Miscellaneous
-    "Scorpion Journal Entry": Spelunky2LocationData(117, "Temple"), # Can be obtained earlier, but Temple is most likely since it has so many pots
-    "Bee Journal Entry": Spelunky2LocationData(118, "Jungle"),
-    "Queen Bee Journal Entry": Spelunky2LocationData(119, "Jungle"),
+    "Scorpion Journal Entry": Spelunky2LocationData(117, "Any World 2"), # Found in the Moon Challenge. Typically farmed from Tiamat in All Journal Entires speedruns
+    "Bee Journal Entry": Spelunky2LocationData(118, "Jungle"), # Also from destroying altars or Temple
+    "Queen Bee Journal Entry": Spelunky2LocationData(119, "Jungle"), # Also from Temple
     "Scarab Journal Entry": Spelunky2LocationData(120, "Dwelling"),
-    "Golden Monkey Journal Entry": Spelunky2LocationData(121, "Menu"),
-    "Leprechaun Journal Entry": Spelunky2LocationData(122, "Menu"),  # Can be in either Jungle or Volcana
+    "Golden Monkey Journal Entry": Spelunky2LocationData(121, "Any World 2"), # Can be found in Dwelling, but is rare

```

Comment:
Could be as early as Dwelling if you get Idol + Altar. But agree with setting it to World 2